src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritProjectListUpdater.java

```diff
@@ -220,7 +220,7 @@ private void tryLoadProjectList() {
             }
         }
         try {
-            logger.info("Trying to load project list.");
+            logger.trace("Trying to load project list.");
```

Comment:
You are right. The only one I care about is the 2nd one...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        From: Jiří EngelthalerSent: Friday, December 4, 2015 07:20To: jenkinsci/gerrit-trigger-pluginReply To: jenkinsci/gerrit-trigger-pluginCc: Scott HebertSubject: Re: [gerrit-trigger-plugin] Fix problem where approvals were not displayed in Query and Trigger (#272)In src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritProjectListUpdater.java:
> @@ -220,7 +220,7 @@ private void tryLoadProjectList() {
>              }
>          }
>          try {
> -            logger.info("Trying to load project list.");
> +            logger.trace("Trying to load project list.");

I like it too but if I have set project list refresh interval to 60 seconds, the log is full of this information. This information is not an error or warning. Admin can create new Log Recoreder and set Logger to com.sonyericsson.hudson.plugins.gerrit or adjust loglevel verbosity for com.sonyericsson.hudson.plugins.gerrit in global log.
But OK, I will remove it from this PR.

—Reply to this email directly or view it on GitHub.